tower-rate-limit/src/error.rs

```diff
@@ -0,0 +1,18 @@
+use std::error;
+
+pub(crate) type Error = Box<error::Error + Send + Sync>;
```

Comment:
I dont feel like this is the style we used in other crates is it?

Reply:
How so? Looks similar to me...

Reply:
What style?

Reply:
the `pub(crate)` but it seems inflight its private. its a non issue was just curious if we maybe wanted to allow these error trait aliases to be avail to users of the crate.

Reply:
We've been keeping the `Box<Error>` alias private. Rustdoc will expand it anyway.

Reply:
works for me 👍